### PR TITLE
Added package target

### PR DESCRIPTION
**Pull request by tomaszrybicki**
Signed-off-by: Tomasz Rybicki <tomasz.rybicki@intel.com>

### CODE REVIEW

File: package.cmake [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"/;/usr/local;/usr/local/include;/usr/local/lib;/run;/var;/var/lib;/lib;/lib/modules;/lib/modules/${UNAME_R};/lib/modules/${UNAME_R}/extra")
+set(CPACK_PACKAGE_VERSION ${VERSION})
+set(CPACK_PACKAGE_NAME ${rpmName})
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_VENDOR "Intel Corporation")
+set(CPACK_PACKAGE_CONTACT "https://github.com/Open-CAS")
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(CPACK_PACKAGE_FILE_NAME
+ "${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-${CPACK_PACKAGE_RELEASE}.${CMAKE_SYSTEM_PROCESSOR}")
+set(CPACK_GENERATOR "RPM;DEB")
```
Comment (tomaszrybicki) [inline package.cmake]: For now there is no switch to determine RPM or DEB, both are built.

File: package.cmake [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Build rpm/deb package using CPack included with CMake
+set(VERSION "20.3")
+set(rpmName "iotrace")
+
+# RPM and DEB only
+set(CPACK_RPM_COMPONENT_INSTALL ON)
+set(CPACK_DEB_COMPONENT_INSTALL ON)
```
Comment (tomaszrybicki) [inline package.cmake]: DEB build needs to be validated.

File: doc/man/CMakeLists.txt [hunk 3]
```diff
@@ -12,9 +12,9 @@ add_custom_command(
 )
 
 install(
-	FILES ${CMAKE_CURRENT_BINARY_DIR}/iotrace.8.gz
+	FILES ${CMAKE_CURRENT_SOURCE_DIR}/MAN.md
```
Comment (mariuszbarczak) [inline doc/man/CMakeLists.txt]: We decided to remove MAN

File: package.cmake (hunk 1 above)
Comment (mariuszbarczak) [reply package.cmake]: let's keep as it is. package can generate both

File: doc/man/CMakeLists.txt (hunk 3 above)
Comment (tomaszrybicki) [reply doc/man/CMakeLists.txt]: This line here was mistakingly changed. I will revert it.
I kept man in normal "make install" installation process.
The package does not install man.

File: package.cmake [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Build rpm/deb package using CPack included with CMake
+set(VERSION "20.3")
```
Comment (mariuszbarczak) [inline package.cmake]: Read version from project

File: doc/man/CMakeLists.txt [hunk 5]
```diff
@@ -12,9 +12,9 @@ add_custom_command(
 )
 
 install(
-	FILES ${CMAKE_CURRENT_BINARY_DIR}/iotrace.8.gz
+	FILES ${CMAKE_CURRENT_SOURCE_DIR}/iotrace.8.gz
```
[the remark this answers is not in the document]
Comment (tomaszrybicki) [reply doc/man/CMakeLists.txt]: Reverted

**Comment by tomaszrybicki** [comment]
This PR partly depends on newest OCTF master

**Pull request merged**